cirq-core/cirq/sim/simulation_state.py

```diff
@@ -100,7 +100,13 @@ def __init__(
     def prng(self) -> np.random.RandomState:
         return self._prng
 
-    def measure(self, qubits: Sequence['cirq.Qid'], key: str, invert_mask: Sequence[bool]):
+    def measure(
+        self,
+        qubits: Sequence['cirq.Qid'],
+        key: str,
+        invert_mask: Sequence[bool],
+        confusion_map: Dict[Tuple[int, ...], np.ndarray],
+    ):
```

Comment:
Good point. The only Cirq code that calls this is `MeasurementGate._act_on_`, but there's still some risk:
1. External measurement gates don't pass this argument, and would break
2. External simulators which reimplement this method won't expect this argument, and would break

(2) actually seems more likely to me than (1), but we can guard against both. I'll put up another PR and mark this one as breaking.